app/routes/jobs.routes.js

```diff
@@ -58,6 +58,20 @@ const routes = [
         crumb: false
       }
     }
+  },
+  {
+    method: 'POST',
+    path: '/jobs/return-logs',
```

Comment:
TBH, I'm a little uncomfortable with a feature such as this having a default cycle. Ideally, I'd like what cycle to be run to be explicit in the request.

Pragmatically, if I can avoid a payload when it comes to the Jenkins job that would also be handy.

So, I know this is a change to how I said to go, but could we change the route to be `jobs/return-logs/${cycle}`?

That way if someone pings a request without one it will hit a 404, which acts as an alternate form of validation. It also acts as a means of documenting that the job is cycle based.

We'll keep the `licenceRef` in the payload for now. Once we're happying this job is working correctly we'll be dropping it anyway.